src/views/under-maintenance/under-maintenance.view.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+
+import useUnderMaintenanceStyles from "src/views/under-maintenance/under-maintenance.styles";
+import { ReactComponent as PolygonZkEVMLogo } from "src/assets/polygon-zkevm-logo.svg";
+import Typography from "src/views/shared/typography/typography.view";
+import routes from "src/routes";
+import { providerError } from "src/adapters/error";
+
+const UnderMaintenance: FC = () => {
+  const classes = useUnderMaintenanceStyles();
+  const navigate = useNavigate();
+  const { state } = useLocation();
+
+  const parsedProviderError = providerError.nullable().safeParse(state);
+
+  const renderProviderError = () => {
+    if (parsedProviderError.success && parsedProviderError.data) {
+      return `${parsedProviderError.data} testnet node`;
+    }
+    return "L1 or L2 testnet nodes";
```

Comment:
We should not hardcode anything about `testnet` in the code. Otherwise, tomorrow when we deploy the mainnet we will need to change this literal.

I propose leaving these messages like this:

*  We can not connect to the Ethereum node
*  We can not connect to the Polygon zkEVM node
*  We cannot connect to the node on L1 or L2

Notice that I changed `with` with `to` and I've slightly changed the literal when we can't decode `parsedProviderError`.

What do you think?